packages/aml-checklist/src/core/connect-session.ts

```diff
@@ -0,0 +1,10 @@
+import { ReapitConnectBrowserSession } from '@reapit/connect-session'
+// Referencing config direct rather than via window because window.reapit.config not available at compile time
+// and throws a runtime error
+import config from '../../config.json'
+
+// Needs to be a singleton as the class is stateful
+export const reapitConnectBrowserSession = new ReapitConnectBrowserSession({
```

Comment:
this one can rewrite like this
```
export const createReapitConnectBrowserSession = () => new ReapitConnectBrowserSession({
  connectClientId: window.reapit.config.cognitoClientId,
  connectOAuthUrl: window.reapit.config.cognitoOAuthUrl,
})
```

Reply:
As discussed @tanphamhaiduong the class needs to be instantiated once only as it's stateful. By wrapping in a function we instantiate each time it is required which is needlessly expensive. Agree importing the config not ideal but it's the best I can come up with at the moment without a compromise elsewhere...